pages/news/hokiport.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Fossil Gen 6 support
+layout: news
+posted: 2023-05-XX
+post: true
+thumbnails: hoki
+summary: The Fossil Gen 6, codenamed "hoki" is now supported by AsteroidOS.
+author: Darrel Griët (MagneFire)
+---
+
+AsteroidOS has been ported to the Fossil Gen 6!
+
+This port is quite unique as it's the first Snapdragon Wear 4100 watch to gain AsteroidOS support.
+
+The port isn't perfect out of the box. Check our [GitHub repository](https://github.com/AsteroidOS/meta-smartwatch/issues?q=is%3Aissue+is%3Aopen+label%3Ahoki) for the current issues.
+
+Most of the currently supported watches can't be bought new. This brought up the desire to port AsteroidOS to newer watches. Breaking new grounds with this sort of ports also helps us gain knowledge useful for future ports.
```

Comment:
Maybe a bit better flowing would be
`Breaking new grounds with this sort of ports helps us to gain useful knowledge for future ports.`

Reply:
This was also copied.